app/src/main/java/com/example/dynamicdiet/service/DietService.kt

```diff
@@ -5,8 +5,8 @@ class DietService {
      * @param userId id of the user
      * @return Diet object
      */
-    suspend fun getDietInfo(var userId: Int): Diet {
-        return Diet('Keto Diet', 2000);
+    suspend fun getDietInfo(userId: Int): Diet {
+        return Diet("Keto Diet", 2000); // Changes quotes
```

Comment:
Changed from single quotes to double quotes
https://kotlinlang.org/docs/coding-conventions.html#class-headers